storagenode/gracefulexit/chore.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gracefulexit
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"go.uber.org/zap"
+	"gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/internal/sync2"
+	"storj.io/storj/storagenode/satellites"
+)
+
+var mon = monkit.Package()
+
+// Chore checks for satellites that are being exited and creates a worker per satellite to complete the process.
+//
+// architecture: Chore
+type Chore struct {
```

Comment:
nit: usually we have order like: log, dependencies, DBs, config flags, other things.

Reply:
updated. are these changes what you had in mind?

Reply:
yes, thanks! Can you also reorganize `NewChore` order?

Reply:
Oops. Updated